packages/web/src/ui/views/viewer.js

```diff
@@ -91,49 +86,46 @@ const viewer = (state, i18n) => {
       })
 
     const doRotatePanZoom = () => {
-      let changed = false
-
       if (rotateDelta[0] || rotateDelta[1]) {
         const updated = orbitControls.rotate({ controls, camera, speed: rotateSpeed }, rotateDelta)
         rotateDelta = [0, 0]
         controls = { ...controls, ...updated.controls }
-        changed = true
+        updateView = true
       }
 
       if (panDelta[0] || panDelta[1]) {
         const updated = orbitControls.pan({ controls, camera, speed: panSpeed }, panDelta)
         panDelta = [0, 0]
         camera.position = updated.camera.position
         camera.target = updated.camera.target
-        changed = true
+        updateView = true
       }
 
       if (zoomDelta) {
         const updated = orbitControls.zoom({ controls, camera, speed: zoomSpeed }, zoomDelta)
         controls = { ...controls, ...updated.controls }
         zoomDelta = 0
-        changed = true
+        updateView = true
       }
 
       if (zoomToFit) {
-        controls.zoomToFit.tightness = 1.3
+        controls.zoomToFit.tightness = 1.5
         const updated = orbitControls.zoomToFit({ controls, camera, entities: prevEntities })
         controls = { ...controls, ...updated.controls }
         zoomToFit = false
-        changed = true
+        updateView = true
       }
-      return changed
     }
 
     // the heart of rendering, as themes, controls, etc change
     const updateAndRender = (timestamp) => {
-      if (doRotatePanZoom() || controls.autoRotate.enabled) {
-        moveRender()
-      }
+      doRotatePanZoom()
 
-      if (renderUntil > Date.now()) {
+      if (updateView || controls.autoRotate.enabled) {
         const updated = orbitControls.update({ controls, camera })
         controls = { ...controls, ...updated.controls }
+        updateView = controls.changed // for elasticity in rotate / zoom
```

Comment:
that was the missing part :) I could not figure out

Reply:
i was lucky to find it!

i knew there had to be something as the rotation did not function properly. but if i clicked 'auto zoom' a few times, the rotation continued. then, i understood that orbitcontrols.update() was adjusting the camera position. BINGO!!!

thanks a lot for the assistance!

Reply:
P.S. I have another fix coming. You will like this one!